Client/Frontend/Fakespot/FakespotViewModel.swift

```diff
@@ -19,19 +19,26 @@ class FakespotViewModel {
             case .loading:
                 elements = [.loadingView]
 
-            case .loaded:
-                elements = [.reliabilityCard,
+            case let .loaded(product):
+                let baseElements = [
+                    ViewElement.reliabilityCard,
                     .adjustRatingCard,
                     .highlightsCard,
                     .qualityDeterminationCard,
-                    .settingsCard]
+                    .settingsCard
+                ]
+                if product?.productId == nil {
+                    elements = [.productCannotBeAnalyzed] + baseElements
```

Comment:
Base elements are only two cards: `[.qualityDeterminationCard, .settingsCard]` (see Figma). It's basically the same as the error state. Any chance we can get the state `error` in this case instead?

Reply:
Sadly error means it was a problem with the API, in this case the call is with succes, but the indicator that product cannot be analysed is the nil id. A bit strange, but we can discuss with desktop.

Reply:
Ok. However base elements should still be only the two cards mentioned above when any message card is displayed.

Reply:
Oh, I see what this is about now. Updated.